egs2/ml_superb/asr2/conf/tuning/train_discrete_asr_e_branchformer1_1gpu_lr5e-4_warmup5k.yaml

```diff
@@ -0,0 +1,93 @@
+# Trained with A100 (40 GB) x 1 GPUs for Kmeans1K+nbpe5K. It takes 32 minutes per epoch.
```

Comment:
Hi, since the two configs are very similar, we may simply keep one with the best performance